tools/mkimage-raw-bios/make-bios

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+
+set -e
+
+set -e
+# for debugging
+[ -n "$DEBUG" ] && set -x
+
+IMGFILE=$PWD/disk.img
+
+
+# we want everything except the final result to stderr
+( exec 1>&2;
+
+ESP_FILE=$PWD/boot.img
+
+
+mkdir -p /tmp/bios
+cd /tmp/bios
+
+# input is a tarball of filesystem on stdin with kernel in /boot
+# output is an iso on stdout
+
+# extract. BSD tar auto recognises compression, unlike GNU tar
+# only if stdin is a tty, if so need files volume mounted...
+[ -t 0 ] || bsdtar xzf -
+
+INITRD="$(find . -name '*.img')"
+KERNEL="$(find . -name kernel -or -name bzImage)"
```

Comment:
we can drop the `bzImage` now - the rename was months ago.

Reply:
Good point. Cleaning up.